.github/workflows/release-bump-as-requirement.yml

```
@@ -0,0 +1,86 @@
+name: 'Maintenance: Bump Action Scheduler version'
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Action Scheduler version (defaults to latest in repository)'
+        required: false
+        default: ''
+
+jobs:
+  bump-version:
+    name: Bump Action Scheduler version
+    runs-on: ubuntu-latest
+    steps:
+    - name: Fetch Action Scheduler version
+      id: get-as-version
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        INPUT_VERSION: ${{ inputs.version }}
+      run: |
+        if [ -z "$INPUT_VERSION" ]; then
+          VERSION=$(gh release view --repo "woocommerce/action-scheduler" --json tagName -q .tagName)
+        else
+          if gh release view "$INPUT_VERSION" --repo "woocommerce/action-scheduler" > /dev/null 2>&1; then
+            VERSION="$INPUT_VERSION"
+          else
+            echo "::error::Action Scheduler version '$INPUT_VERSION' is not valid."
+            exit 1
+          fi
+        fi
+
+        echo "version=$VERSION" >> $GITHUB_OUTPUT
+    - name: Check out trunk
+      uses: actions/checkout@v4
+      with:
+        ref: trunk
+    - name: Set up dev environment
+      run: |
+        cd plugins/woocommerce
+
+        # Install dev packages. Required for changelogger use.
+        composer install --quiet
+
+        # Configure Git.
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+    - name: Bump Action Scheduler requirement
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd plugins/woocommerce
+
+        # Update A-S version.
+        jq --tab -r ".require[\"woocommerce/action-scheduler\"] |= \"${{ steps.get-as-version.outputs.version }}\"" composer.json > composer.json.new
+        mv composer.json.new composer.json
+
+        if git diff --quiet; then
+          echo "::error::No changes to commit."
+          exit 1
+        fi
+
+        composer update woocommerce/action-scheduler --no-scripts
+
+        branch_name="update-action-scheduler-${{ steps.get-as-version.outputs.version }}"
```

Small nitpick to match our branch naming conventions:

```suggestion
        branch_name="update/action-scheduler-${{ steps.get-as-version.outputs.version }}"
```

Just sanity checking this: [it failed on the second workflow](https://github.com/joshuatf/woocommerce/actions/runs/16317583463/job/46087098961) due to the branch already existing.  We shouldn't have this problem in this repo though since branches auto-delete after merge.

---

Thanks Josh. Fixed in e7cdce4916e74960957925a06b05feb1dacfbef8.